testing/scripts/k8s_utils.py

```diff
@@ -3,6 +3,8 @@
 import subprocess
 import os
 import time
+import random
+from retrying import retry
```

Comment:
Use of retry is to handle sporadic connectivity failures. See https://github.com/SeldonIO/seldon-core/issues/473